src/common/LogEntry.h

```diff
@@ -187,6 +191,8 @@ inline std::ostream& operator<<(std::ostream& out, const clog_type t)
   }
 }
 
+template <> struct fmt::formatter<clog_type> : fmt::ostream_formatter {};
```

Comment:
why not directly where that type was declared, i.e. in ostream_temp.h?

Reply:
I did it here for code locality reasons, as the ostream impl is [just above](https://github.com/ceph/ceph/pull/53698/files/53363c65a938b6f0854c4b5f23890f9289a63ec1#diff-ecb790fabfca8053e26d2e6483a62314e6db02c401ff8ab0204a84fc02807db3R176-R192). I can move it, but that will require `#include`ing fmtlib in common/ostream_temp.h and dependents... is that acceptable?